railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -70,9 +70,19 @@ RUN apt-get update -qq && \
 COPY --from=build /usr/local/bundle /usr/local/bundle
 COPY --from=build /rails /rails
 
+# add custom user
+RUN useradd ruby
+
+USER ruby:ruby
+
+COPY --from=builder --chown=ruby:ruby /usr/local/bundle /usr/local/bundle
```

Comment:
It's totally wasteful to chown in another layer.

Reply:
you mean wasteful to have 2 chowns in 2 copy commands?

Reply:
also updated, agree here, didn't notice it in the first place